patches/minecraft/net/minecraft/world/entity/LivingEntity.java.patch

```diff
@@ -392,6 +417,29 @@
              if (map == null) {
                 map = Maps.newEnumMap(EquipmentSlot.class);
              }
+@@ -2467,7 +_,8 @@
+          double d7;
+          if (this.m_20077_()) {
+             d7 = this.m_20120_(FluidTags.f_13132_);
+-         } else {
++         } else if (this.isInFluid()) d7 = this.getFluidHeight(this.getTouchingFluid());
++         else {
```

Comment:
Again the patch could be slightly smaller if it was:
```
} else if (thing) {
body
} else {
```
or maybe given the extra line doesn't really gain much in readability here.
```
} else if (thing) { body
} else {
```

Reply:
No Pup, the body of elses should not be on the same line. The rules about patch sizes are for clarity and simplicity. Not a hard line number. He should move the body to a new line, and read the braces.